src/HDInsight/HDInsight/ManagementCommands/GetAzureHDInsightPropertiesCommand.cs

```diff
@@ -20,9 +20,8 @@
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
-    [GenericBreakingChange("Get-AzHDInsightProperties alias will be removed in an upcoming breaking change release", "2.0.0")]
+    [CmdletOutputBreakingChange(typeof(CapabilitiesResponse), ReplacementCmdletOutputTypeName = "AzureHDInsightCapabilities")]
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "HDInsightProperty"),OutputType(typeof(CapabilitiesResponse))]
-    [Alias("Get-AzHDInsightProperties")]
```

Comment:
> Same comment. You cannot remove this alias until a breaking change release. If you want to remove the alias at the next breaking change release in November, then leave this alias and the breaking change attribute here and you can remove it then.
> 
> Otherwise remove the old breakign change attribute and leave the alias.
> 
> This is the other thing that is causing static analysis failures, which prevent us from merging this PR.

Thanks. The same with above.